### ISSUE TITLE
Strange results

### ISSUE THREAD

**Issue by yousef-xailient** [sara-nl/attention-sampling-pytorch#1]
Hi there,

Thank you for providing a PyTorch implementation of the original!

I have cloned your repo and tried to run the experiment on the speed limit sign dataset and I am getting strange results not close to what is shown in the paper.

I have merely cloned the repo, ran your speed limit dataset script to download the data, and speed_limit.py to run the job.

Any help would be appreciated!

![Screenshot from 2020-08-01 15-26-54](https://user-images.githubusercontent.com/57377547/89094841-a408f280-d40b-11ea-89df-18b6465bbbaf.png)

**Comment by jorism1993** [comment]
Hallo,

Thank you for emailing me. I am on vacation at the moment and am therefore not to investigate it further. I will try to answers your question once I am back.

Kind regards,

Joris

________________________________
Van: yousef-xailient <notifications@github.com>
Verzonden: zaterdag 1 augustus 2020 07:29
Aan: sara-nl/attention-sampling-pytorch <attention-sampling-pytorch@noreply.github.com>
CC: Subscribed <subscribed@noreply.github.com>
Onderwerp: [sara-nl/attention-sampling-pytorch] Strange results (#1)


Hi there,

Thank you for providing a PyTorch implementation of the original!

I have cloned your repo and tried to run the experiment on the speed limit sign dataset and I am getting strange results not close to what is shown in the paper.

I have merely cloned the repo, ran your speed limit dataset script to download the data, and speed_limit.py to run the job.

Any help would be appreciated!

[Screenshot from 2020-08-01 15-26-54]<https://user-images.githubusercontent.com/57377547/89094841-a408f280-d40b-11ea-89df-18b6465bbbaf.png>

**Comment by jorism1993** [comment]
Hello,

You might need to do some longer training. I included a screenshot of my tensorboard logs from the traffic sign recognition tasks.

[cid:528897b4-b536-44ab-a1ef-60f9cfe7de65]

I hope you get it too work! If you have any questions in the future, feel free to contact me.

Kind regards,

Joris

________________________________
Van: yousef-xailient <notifications@github.com>
Verzonden: zaterdag 1 augustus 2020 07:29
Aan: sara-nl/attention-sampling-pytorch <attention-sampling-pytorch@noreply.github.com>
CC: Subscribed <subscribed@noreply.github.com>
Onderwerp: [sara-nl/attention-sampling-pytorch] Strange results (#1)


Hi there,

Thank you for providing a PyTorch implementation of the original!

I have cloned your repo and tried to run the experiment on the speed limit sign dataset and I am getting strange results not close to what is shown in the paper.

I have merely cloned the repo, ran your speed limit dataset script to download the data, and speed_limit.py to run the job.

Any help would be appreciated!

[Screenshot from 2020-08-01 15-26-54]<https://user-images.githubusercontent.com/57377547/89094841-a408f280-d40b-11ea-89df-18b6465bbbaf.png>

**Comment by jingweizhang-xyz** [comment]
Hi,

I am experimenting the same issue here. I tried MegaMNIST with no problem but had the same problem on speed limit dataset. I can not see the screenshot of your tensorboard logs. Could you please share it or tell me how many epochs is required? I really appreciate it.

**Comment by jorism1993** [comment]
Hi,

For the speed limit dataset I am running for a 1000 epochs. You can download the tensorboard log file [here][https://we.tl/t-Sf0IrRDcJJ]. 

Kind regards,

Joris


________________________________
Van: Jingwei Zhang <notifications@github.com>
Verzonden: woensdag 30 september 2020 06:36
Aan: sara-nl/attention-sampling-pytorch <attention-sampling-pytorch@noreply.github.com>
CC: jorism1993 <jorismollinga@hotmail.com>; Comment <comment@noreply.github.com>
Onderwerp: Re: [sara-nl/attention-sampling-pytorch] Strange results (#1)


Hi,

I am experimenting the same issue here. I tried MegaMNIST with no problem but had the same problem on speed limit dataset. I can not see the screenshot of your tensorboard logs. Could you please share it or tell me how many epochs is required? I really appreciate it.

**Comment by jingweizhang-xyz** [comment]
Thank you! I'll have a try again.

**Comment by zuh17h** [comment]
@skykiny I have a similar problem! Did you get better results with increasing the number of epochs?

**Comment by zuh17h** [comment]
Can I ask what is the best accuracy that you get? I cannot access the log file.
